translate/convert/test_web2py2po.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+import sys
+
+from translate.convert import web2py2po
+from translate.storage import po
+from translate.storage.test_base import first_translatable, headerless_len
+
+
+class TestWEB2PY2PO(object):
+
+    def web2py2po(self, web2py_source):
+        """helper that converts po source to web2py source without requiring files"""
+        input_web2py = eval(web2py_source)
+        new_pofile = po.pofile()
+        convertor = web2py2po.web2py2po(new_pofile)
+        output_po = convertor.convertstore(input_web2py)
+        return output_po
+
+    def singleelement(self, storage):
+        """checks that the pofile contains a single non-header element, and returns it"""
+        assert headerless_len(storage.units) == 1
+        return first_translatable(storage)
+
+    def test_basic(self):
```

Comment:
I actually don't know anything about this format, but I would be a bit surprised if this is the only test needed to cover all scenarios involved in dealing with this format conversions. Can you please add some additional tests? Thank you very much in advance.

Reply:
Same for the other test file above.

Reply:
Sure thing!  Really, only a test regarding Unicode characters is needed -- both this test and the one I just mentioned covers all the scenarios this format will really present.